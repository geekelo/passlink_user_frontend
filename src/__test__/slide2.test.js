import React from 'react';
import { render } from '@testing-library/react';
import SlideTwo from '../homePage/javascripts/slide2';

test('renders SlideTwo correctly', () => {
  const { asFragment } = render(<SlideTwo />);
  
  // Initial render snapshot
  expect(asFragment()).toMatchSnapshot();
});
