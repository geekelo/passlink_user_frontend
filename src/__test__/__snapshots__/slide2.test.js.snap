// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders SlideTwo correctly 1`] = `
<DocumentFragment>
  <div
    class="body-slide"
  >
    <div
      class="slide-content"
    >
      <h1
        class="title"
      >
        An effective event management portal.
      </h1>
      <div
        class="ticks-cta-section"
      >
        <div
          class="ticks-section"
        >
          <p>
            <span
              class="ticks"
            >
              ✔
            </span>
            <span>
              Get insights and analytics on event attendees
            </span>
          </p>
          <p>
            <span
              class="ticks"
            >
              ✔
            </span>
            <span>
              Get feedback on your private and public events
            </span>
          </p>
          <p>
            <span
              class="ticks"
            >
              ✔
            </span>
            <span>
              Let others know about your events & Limit access to registered attendees
            </span>
          </p>
          <p>
            <span
              class="ticks"
            >
              ✔
            </span>
            <span>
              Send reminders to registered attendees
            </span>
          </p>
        </div>
        <button
          class="btn"
          type="submit"
        >
          <a
            href="v.com"
          >
            <span>
              ADD EVENT  
            </span>
            <i
              class="material-icons"
            >
              event
            </i>
          </a>
        </button>
      </div>
    </div>
    <div
      class="infographic-section"
    >
      <img
        alt="infographic"
        class="infographic"
        src="infographic2.png"
      />
    </div>
  </div>
</DocumentFragment>
`;
